crates/modelardb_server/src/storage/mod.rs

```diff
@@ -74,17 +74,18 @@ use crate::storage::uncompressed_data_manager::UncompressedDataManager;
 use crate::PORT;
 
 /// The folder storing uncompressed data in the data folders.
-pub const UNCOMPRESSED_DATA_FOLDER: &str = "uncompressed";
+const UNCOMPRESSED_DATA_FOLDER: &str = "uncompressed";
 
 /// The folder storing compressed data in the data folders.
-pub const COMPRESSED_DATA_FOLDER: &str = "compressed";
+pub(super) const COMPRESSED_DATA_FOLDER: &str = "compressed";
 
 /// The scheme with host at which the query data folder is stored.
-pub const QUERY_DATA_FOLDER_SCHEME_WITH_HOST: &str = "query://query";
+pub(super) const QUERY_DATA_FOLDER_SCHEME_WITH_HOST: &str = "query://query";
 
 /// A static UUID for use in tests. It is not in a test utilities module so it can be used both
-/// inside the if cfg(test) block of [`create_time_and_value_range_file_name`] and in test modules.
-pub const TEST_UUID: Uuid = uuid!("44c57d06-333c-4935-8ae3-ed7bc53a08c4");
+/// inside the if cfg(test) block of [`StorageEngine::create_time_and_value_range_file_name`] and in
```

Comment:
Maybe change to `StorageEngine::create_time_and_value_range_file_name()` to make it clear that it is a function. This would also be consistent with the doc comment for this file.

Reply:
`StorageEngine::` is needed as `create_time_and_value_range_file_name()` is currently a method in `StorageEngine` that simply does not take `self` as as an argument.

Reply:
I removed the old comment and added this one that just says to add `()`. I realized `StorageEngine` was needed so I deleted it.

Reply:
Fixed.